### DISCUSSION TITLE
Daily 03/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#92] [General]
El viernes estuve estudiando Html y Css.
El fin de semana continue con el estudio de CSS y HTML. Rendí el exámen de Full Stack. Participe en la sprint planing. Pase a issues las US que quedaban pendientes de la documentación. y también estuve seleccionando tareas para realizar durante el Sprin 1
Hoy no conte con mucho tiempo. Estuve estudiando las guías de Gestíon de Proyecto en la plataforma de Ispc.

**Comment by rpgrca** [comment]
Convertí a PDF la última versión de la ERS durante el fin de semana y hoy no hice nada, por eso no armé daily, se me pasó. Digamos que hoy no fui a trabajar. Mañana pienso seleccionar algunas tareas para no ir atrasándome.

**Comment by Thiagox749** [comment]
El Viernes repasé ultimos contenidos para el examen de valoracion, HTML y CSS, el fin de semana realicé dicho examen y terminé el modulo 4 de python. Hoy repasé los contenidos de las guías de Gestion de Proyectos para proxima valoración. Mañana veré las tareas a realizar para avanzar en el proyecto.
